### PR TITLE
Update bitcoin-abc@0.20.2

### PR DESCRIPTION
**Pull request by adisney-up**
Most significant changes from Dockerfiles from previous versions is the standardization and improvement to pgp and checksum verification for the tar files.

### CODE REVIEW

**Comment by joaopaulofonseca** [comment]
Since this is still open, can you please update this to use the latest version, 0.20.4?

File: 0.20/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+RUN curl -o /usr/local/bin/gosu -fSL https://github.com/tianon/gosu/releases/download/${GOSU_VERSION}/gosu-$(dpkg --print-architecture) \
+  && curl -o /usr/local/bin/gosu.asc -fSL https://github.com/tianon/gosu/releases/download/${GOSU_VERSION}/gosu-$(dpkg --print-architecture).asc \
+  && gpg --verify /usr/local/bin/gosu.asc \
+  && rm /usr/local/bin/gosu.asc \
+  && chmod +x /usr/local/bin/gosu
+
+ENV BITCOIN_ABC_DATA=/home/bitcoin/.bitcoin
+ENV BITCOIN_ABC_VERSION=0.20.2
+ENV BITCOIN_ABC_TAR=bitcoin-abc-${BITCOIN_ABC_VERSION}-x86_64-linux-gnu.tar.gz
+ENV BITCOIN_ABC_ASC=jasonbcox-sha256sums.${BITCOIN_ABC_VERSION}.asc
```
Comment (joaopaulofonseca) [inline 0.20/Dockerfile]: Awesome. I like this standardization! 👍

**Comment by ianeinser** [comment]
Hi,
when is the image of 0.20.4 gonna available for download? Thanks

**Comment by joaopaulofonseca** [comment]
Hi @ianeinser, we expect this to be released in the coming days.

**Pull request merged**